### PR TITLE
[Feature] Rebranding

### LINKED ISSUE #3575: Rebranding logos
Hi team we need to apply the new logos and icons for the next sections:

- [ ] Wazuh icon in the Kibana menu
- [ ] Wazuh logo in Wazuh menu
- [ ] Wazuh logo in Health-check
- [ ] Wazuh logo in reports
- [ ] Default background in login page (cloud version)
- [ ] Wazuh logo in login page (cloud version)
- [ ] Check color palette

### PR DESCRIPTION
**Pull request by matiasmoreno876**
Hi team, in this PR I merged, the work done by @yenienserrano in the [PR](https://github.com/wazuh/wazuh-kibana-app/pull/3666). Changes are for version 4.3

Closes #3575

### CODE REVIEW

File: public/styles/component.scss [hunk 1]
```diff
@@ -118,4 +118,8 @@ kbn-dis doc-table .kbnDocViewer__warning {
 }
 .header__breadcrumbsWithExtensionContainer .header__breadcrumbsAppendExtension {
     flex-grow: 0;
+}
+
+.euiCard .euiCard__top .euiIcon--app .euiIcon__fillSecondary{
+    fill: #256BD1
```
Comment (Desvelao) [inline public/styles/component.scss:124]: **thought**: This style could be affecting some components of Kibana apart from the Wazuh app. We should restrict this to the Wazuh app.

**Comment by Desvelao** [comment]
**review**:

In dark mode, the logo doesn't look so good.
image
image

Should we replace it depending on the selected theme?

File: public/styles/component.scss (hunk 1 above)
Comment (Desvelao) [reply public/styles/component.scss:124]: **thought**: Was the secondary color changed? In [this comment](https://github.com/wazuh/wazuh-kibana-app/issues/3575#issuecomment-951966154), the color seems to be `#3585F9`. Could we be sure that color we should apply?

**Comment by Desvelao** [comment]
## Changes
- Apply brand images depending on app configuration with custom (no themes) or default (with themes)
- Changed default values of `customization` app setting to empty.
  - `customization.logo.app`
  - `customization.logo.sidebar`
  - `customization.logo.healthcheck`
  - `customization.logo.reports`
- Moved the app screenshots images to `screenshots` directory
- Organize the default assets in the next structure:
  - fonts
  - images
    - icons: moved the icons (gmail)
    - themes: moved the default brand images. Added images for dark theme.
- Moved some values to `contants.ts` file
- Created functions related to the assets that create the path of them
- Add logos (line art) for Docker, GitHub, Google Cloud, Office 365 as ReactJS components
- Replaced logos for Docker, GitHub, Google Cloud and Office 365 in `Modules directory`
- Add app styles for `light` and `dark` themes.
  - Created mixin to apply the brand color in `light` and `dark` themes.
  - Centralized the brand colors using SCSS variables
  - Changed some styles in the health check
- Apply `primary` color for icons in the app menu

### Screenshots
`light` theme:
- Logos
  - Menu and sidebar
  image

  - Health check
  image

  - Report
  image

  - Loading
  image

- Module logos in `Modules directory`
image
image
image
image
image

- Module logo in `Panel` tab
image
image

`dark` theme
- Logos
  - Menu and sidebar
  image

  - Health check
  image

  - Report
  image
  
  - Loading
  image

  - Module logos in `Modules directory`
  image
image
image
image
image

  - Module logo in `Panel` tab
  image
  image

**Comment by yenienserrano** [comment]
Shouldn't that icon be blue?
image
image

**Comment by Desvelao** [comment]
> Shouldn't that icon be blue? image image

This is not related to the PR but it is a good appreciation that we could consider.

The colors of that icon are defined with the `fill` property and they are not altered when setting the `color` property of the button. We could change this with styles.

**Pull request merged**